tests/middleware/views.py

```diff
@@ -1,5 +1,25 @@
 from django.http import HttpResponse
+from django.utils.decorators import classonlymethod
+from django.views.generic import View
 
 
 def empty_view(request, *args, **kwargs):
     return HttpResponse()
+
+
+def sensitive_fbv(request, *args, **kwargs):
+    return HttpResponse()
+
+
+sensitive_fbv._should_append_slash = False
+
+
+class SensitiveCBV(View):
+    def get(self, *args, **kwargs):
+        return HttpResponse()
+
+    @classonlymethod
+    def as_view(cls, **initkwargs):
+        view = super().as_view(**initkwargs)
+        view._should_append_slash = False
```

Comment:
No... because you don't get the class back, you get the wrapping view function, which is the callback the resolver match has at `match.func`. 

(Thinking about the best way to do this if we have a decorator, [refs docs](https://docs.djangoproject.com/en/3.1/topics/class-based-views/intro/#decorating-class-based-views))

Reply:
Ah of course... 👀